### PR TITLE
Prelaunch fixes

### PR DESCRIPTION
**Pull request by touchRED**
Notes: 

- Fixes header order in accordance with semantic guidelines
- Adds title to logo, making it accessible for screenreaders
- Adjusts tabindex for interactive items in the offscreen nav, turning them off when the nav is hidden
- Moves the offscreen nav into the `<nav>` element
- Adds `@reach/skip-nav` functionality
- Other small fixes here and there

### CODE REVIEW

File: src/pages/faq.js [hunk 1]
```diff
@@ -39,24 +39,7 @@ const FaqHero = () => {
...
-              >
-                Asked
-              </motion.h2>
-              <motion.h1
-                variants={basicStaggerChild}
-                className="text-uppercase knockout-bold"
-              >
-                Questions
-              </motion.h1>
+              <h1><span>Frequently</span> <span>Asked</span> <span>Questions</span></h1>
```
Comment (Lane) [inline src/pages/faq.js]: the HTML semantics are good here, but i think James has tweaked these animations to his liking, so we should keep them.

File: src/pages/index.js [hunk 2]
```diff
@@ -349,10 +339,10 @@ const HomeCard = ({ cardData, index }) => {
...
             animate={inView ? 'show' : 'hide'}
           >
-            <motion.h5
+            <motion.h2
               variants={basicStaggerChild}
               className="knockout-bold text-uppercase"
-            >{`${index + 1}. ${cardData.cardTitle}`}</motion.h5>
+            >{`${index + 1}. ${cardData.cardTitle}`}</motion.h2>
             <motion.div variants={basicStaggerChild}>
               <MDXRenderer>{cardData.cardHeading}</MDXRenderer>
```
Comment (Lane) [inline src/pages/index.js]: these are set as heading level 4 in MDX frontmatter.  can we drop the heading 4 markdown in the frontmatter (`####`) and wrap this `MDXRenderer` in an `<h4>`?

File: src/pages/pathways-to-mobility.js [hunk 3]
```diff
@@ -39,24 +39,7 @@ const CasesHero = ({ pageData }) => {
...
-              >
-                To
-              </motion.h2>
-              <motion.h1
-                variants={basicStaggerChild}
-                className="text-uppercase knockout-bold"
-              >
-                Mobility
-              </motion.h1>
+              <h1><span>Pathways</span> <span>To</span> <span>Mobility</span></h1>
```
Comment (Lane) [inline src/pages/pathways-to-mobility.js]: semantics are good, but we should keep animations.

File: src/templates/research-topic.js [hunk 4]
```diff
@@ -162,44 +162,36 @@ const TopicNext = ({ pageData }) => {
...
+                                        filename={nextTopic.item_image}
+                                    />
+                                </div>
+                            }
+                        </div>
+                    </Col>
+                </Row>
+            </Container>
+        </div>
+    )
```
Comment (Lane) [inline src/templates/research-topic.js]: noticing switch to 4 character indent here.  are you using prettier to autoformat?  if so it doesn't seem to be picking up on the `.prettierrc`.

**Review by Lane — CHANGES_REQUESTED** [review]
thanks for getting to these accessibility updates! a couple changes:

- [ ] re-add animations
- [ ] heading tag update on homepage

**Comment by touchRED** [comment]
@Lane not sure why it took me so long to notice that you had requested changes for this one – sorry about that! I believe I've addressed things as you intended – let me know if we're not quite there yet

**Comment by Lane** [comment]
@touchRED sorry for the delay getting to this, can you fix up the conflicts?

**Comment by touchRED** [comment]
@Lane fixed the conflict in `home.mdx` and thought I got the one in `package-lock.json` since my command line is saying everything is up to date, but Github seems to think differently.

Looks like we may need to merge and push master from the command line – wanted to check with you before pushing to master so let me know if you'd like me to do this!

**Comment by Lane** [comment]
@touchRED went ahead and synced the `package-lock.json` and resolved some conflicts that came up from merging #122, going to merge this one up.

**Review by Lane — APPROVED** [review]

**Pull request merged**